kubernetes_deploy/deployment.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: laa-fee-calculator
+spec:
+  template:
+    metadata:
+      labels:
+        app: laa-fee-calculator-app
+    spec:
+      containers:
+      - image: 926803513772.dkr.ecr.eu-west-1.amazonaws.com/claim-for-crown-court-defence/laa-fee-calculator:latest
+        name: app
+        ports:
+        - containerPort: 8080
+          name: http
+        env:
+        - name: DEBUG
+          value: "True"
+        - name: DJANGO_SECRET_KEY
+          value: CHANGE_ME
+        - name: ENV
+          value: local
```

Comment:
Is this the only deployment file? For all environments?

Reply:
I'm asking because `ENV` should be environment specific, `DEBUG` shouldn't be set on prod, and `DJANGO_SECRET_KEY` probably shouldn't be set at all as I believe it's being supplied as a start-up argument.

Reply:
Good point @ibrechin 
Currently, the only thing that changes per environment is the image https://github.com/ministryofjustice/laa-fee-calculator/pull/55/files#diff-1d37e48f9ceff6d8030570cd36286a61R45 but you can also control all of the environment variable using `kubectl set env`